nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,80 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+// added reviews for all meals
+function getAllMealsWithReviews() {
+  meals.map(meal => {
```

Comment:
map returns a new array, if you don't want to use get the array, then it's better to change it with forEach. you can read more about the map here: 
https://www.freecodecamp.org/news/javascript-map-reduce-and-filter-explained-with-examples/

Reply:
Thank you. Yes, that I should fix.